package.json

```diff
@@ -14,41 +14,42 @@
   "dependencies": {
     "@expo-google-fonts/didact-gothic": "^0.1.0",
     "@expo-google-fonts/poppins": "^0.1.0",
-    "@react-native-async-storage/async-storage": "1.15.1",
+    "@react-native-async-storage/async-storage": "^1.13.0",
     "@react-native-community/masked-view": "0.1.10",
     "@react-navigation/native": "5.9.4",
     "@react-navigation/stack": "5.14.4",
     "aes-js": "3.1.2",
     "color": "^3.1.3",
-    "expo": "~40.0.0",
+    "expo": "^41.0.0",
     "expo-app-loading": "^1.0.1",
-    "expo-auth-session": "~3.0.0",
-    "expo-barcode-scanner": "~9.1.0",
-    "expo-random": "~10.0.0",
-    "expo-secure-store": "~9.3.0",
-    "expo-status-bar": "~1.0.3",
-    "expo-web-browser": "~8.6.0",
-    "firebase": "^7.9.0",
+    "expo-auth-session": "~3.2.3",
+    "expo-barcode-scanner": "~10.1.2",
+    "expo-random": "~11.1.2",
+    "expo-secure-store": "~10.1.0",
+    "expo-status-bar": "~1.0.4",
+    "expo-web-browser": "~9.1.0",
+    "firebase": "8.2.3",
     "react": "16.13.1",
     "react-dom": "16.13.1",
-    "react-native": "https://github.com/expo/react-native/archive/sdk-40.0.1.tar.gz",
-    "react-native-gesture-handler": "~1.8.0",
-    "react-native-get-random-values": "1.7.0",
+    "react-native": "https://github.com/expo/react-native/archive/sdk-41.0.0.tar.gz",
+    "react-native-gesture-handler": "~1.10.2",
+    "react-native-get-random-values": "~1.7.0",
     "react-native-paper": "^4.7.2",
-    "react-native-reanimated": "~1.13.0",
-    "react-native-safe-area-context": "3.1.9",
-    "react-native-screens": "~2.15.2",
-    "react-native-svg": "^12.1.0",
+    "react-native-reanimated": "~2.1.0",
+    "react-native-safe-area-context": "3.2.0",
+    "react-native-screens": "~3.0.0",
+    "react-native-svg": "12.1.0",
     "react-native-web": "~0.13.12",
     "url-otpauth": "^2.0.0",
     "uuid": "8.3.2"
   },
   "devDependencies": {
-    "@babel/core": "~7.13.15",
+    "@babel/core": "~7.9.0",
```

Comment:
Same, expo upgrade util downgraded it. We could up it back, but may be it downgraded for some reason!?